.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
     "extends": "girder",
+    "rules": {
+        "no-throw-literal": "off"
```

Comment:
This is a good question, @brianhelba could you link to some docs about the use of `throw undefined` in jquery promises? I am having trouble finding them with google.